components/forms/TransactionSigning.tsx

```diff
@@ -21,15 +23,18 @@ interface LoadingStates {
 interface Props {
   signatures: DbSignature[];
   tx: DbTransaction;
+  pubkey: MultisigThresholdPubkey;
   transactionID: string;
   addSignature: (signature: DbSignature) => void;
 }
 
 const TransactionSigning = (props: Props) => {
+  const memberPubkeys = props.pubkey?.value.pubkeys.map(({ value }) => value);
```

Comment:
```suggestion
  const memberPubkeys = props.pubkey.value.pubkeys.map(({ value }) => value);
```

this is not optional, right?

Reply:
It is undefined when the component loads and then gets a value. Maybe the type is wrong

Reply:
okay, but it seems like you already cover that case in the caller code. See diff in `pages/multi/[address]/transaction/[transactionID].tsx` below